controllers/managedocs_controller.go

```diff
@@ -1057,6 +1067,14 @@ func (r *ManagedOCSReconciler) reconcilePrometheus() error {
 			},
 			Key: alertRelabelConfigSecretKey,
 		}
+		if r.DeploymentType != convergedDeploymentType {
+			if remoteWriteSpec, err := r.getPrometheusRemoteWriteSpec(); remoteWriteSpec != nil {
```

Comment:
Why aren't we acting on the error? It is one thing to skip if we have no configuration and another one to handle errors in case there is configuration

Reply:
I am logging the error. If the configuration is fault shoudln't we just disable remote write. Will propagating  the error to the  reconciler help?

Reply:
If we have fault config I am disabling remote write, will propagating this error to reconciler help?

Reply:
Why? are we doing anything like that anywhere else?.
There is a difference between understanding that we need to skip remote write if it isn't configured, which convey the intent that there isn't a requirement to remote write, and between configured but broken, which in all other cases is in the form of a return of an error

Reply:
As discussed offline, this should block installation of the addon if configurations are not fault or not present.
Fixed in next patch